packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -82,6 +82,43 @@ export const bm25s = (model: ModelData): string[] => [
 retriever = BM25HF.load_from_hub("${model.id}")`,
 ];
 
+export const depth_anything_v2 = (): string[] => {
+	const shellCommands = [
+		`# Install the Depth-Anything-V2 library
+git clone https://github.com/DepthAnything/Depth-Anything-V2
+cd Depth-Anything-V2
+pip install -r requirements.txt`,
```

Comment:
```ts
# Install from https://github.com/DepthAnything/Depth-Anything-V2
```

I would just have this line and that's it to reduce the snippet size (same as MarsTTS example)